### PR TITLE
feat: improve widget customization

### PR DESCRIPTION
**Pull request by chybisov**

### CODE REVIEW

File: packages/widget/src/themes/createTheme.ts [hunk 1]
```diff
@@ -0,0 +1,415 @@
+import { loadingButtonClasses } from '@mui/lab';
```
Comment (DNR500) [inline packages/widget/src/themes/createTheme.ts:1]: I really like having the theme unified and present like this in the one function :-)

File: packages/widget/src/themes/jumper.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import type { WidgetTheme } from '../types/widget.js';
+
+export const jumperTheme: WidgetTheme = {
```
Comment (DNR500) [inline packages/widget/src/themes/jumper.ts:3]: This theme doesn't seem to do a lot - is it still needed?

File: packages/widget/src/themes/windows95.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        root: {
+          backgroundColor: '#fff',
+          boxShadow:
+            'inset -1px -1px #ffffff, inset 1px 1px #0a0a0a, inset -2px -2px #dfdfdf, inset 2px 2px #808080',
+        },
+      },
+    },
+    MuiIconButton: {
+      defaultProps: {
+        // disableRipple: true,
```
Comment (DNR500) [inline packages/widget/src/themes/windows95.ts:82]: Should we need to keep this?

**Review by DNR500 — APPROVED** [review]
This looks good. Couple of questions about if things need deleting but nothing that would stop this being merged in. Thanks Eugene :-)

File: packages/widget/src/themes/jumper.ts (hunk 2 above)
Comment (chybisov) [reply packages/widget/src/themes/jumper.ts:3]: I'll update it later this week 🙂

**Pull request merged**